.gitignore

```diff
@@ -1,2 +1,6 @@
 node_modules
 .env.local
+/monorepo/monorepo-ws/libs/prisma-article-app-client/src/generated
+/monorepo/monorepo-ws/libs/supergraph-manager/src/generated
+/monorepo/monorepo-ws/libs/prisma-user-app-client/src/generated
```

Comment:
**/generated nebo něco podobnýho by mohlo naradit 3 řádky a zároveń ti to bude fungovat i na nový případy

Reply:
nahrazeno